src/opam_create.ml

```diff
@@ -1,29 +1,49 @@
 open Stdune
 
+let default_build_command = lazy (
+  let path = Path.of_string "<internal>" in
+  let dummy_opam =
+      Opam_file.of_string ~path {|
+build: [
+  [ "dune" "subst" ] {pinned}
+  [ "dune" "build" "-p" name "-j" jobs]
```

Comment:
Don't we need:

```
[ "dune" "build" "-p" name "-j" jobs] {with-test}
[ "dune" "build" "-p" name "@doc"] {with-doc}
```

here as well?

Reply:
I will add those in. But really, running tests per package via dune doesn't seem to be well supported. We should reconsider recommending it.